### PR TITLE
Migration/textnote

### PR DESCRIPTION
**Pull request by rvilarl**
TextNote !

### CODE REVIEW

File: src/textnote.ts [hunk 1]
```diff
@@ -6,20 +6,43 @@
 // Examples of this would be such as dynamics, lyrics, chord changes, etc.
 
 import { Vex } from './vex';
-import { Note } from './note';
+import { Note, NoteStruct } from './note';
 import { Glyph } from './glyph';
+import { FontInfo } from './types/common';
+import { StaveNoteStruct } from './stavenote';
```
Comment (0xfe) [inline src/textnote.ts:12]: This is not used, you can remove.

Comment (rvilarl) [reply src/textnote.ts]: done

**Comment by rvilarl** [comment]
Ready!

**Comment by rvilarl** [comment]
@0xfe anything pending here?

**Comment by 0xfe** [comment]
Looks good.

**Pull request merged**